contracts/NodeDelegator.sol

```diff
@@ -207,11 +208,25 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
         ndcAssets += IERC20(asset).balanceOf(address(this));
 
         if (asset == WETH_TOKEN_ADDRESS) {
-            // Add any ETH in the NDC that was earned from EigenLayer
+            // Add any ETH in the NDC that was earned from execution rewards
             ndcAssets += address(this).balance;
 
             eigenAssets = stakedButNotVerifiedEth;
+
+            // Not getting ETH restaked into EigenLayer as that is not yet supported
+            // by the NodeDelegator.
+            // The WETH asset will point to the EigenLayer beaconChainETHStrategy
+            // 0xbeaC0eeEeeeeEEeEeEEEEeeEEeEeeeEeeEEBEaC0
+
+            // Add any consensus rewards that have been sent to the EigenPod.
+            // In the future this can include full validator withdrawals that have been swept
+            // to the EigenPod but native ETH restaking and withdrawals is not currently supported
+            // by the NodeDelegator.
+            if (eigenPod != address(0)) {
```

Comment:
This would double count rewards if a validator were to withdraw, since the validator funds would also be stored in stakedButNotVerifiedEth. We should remove the `eigenAssets += eigenPod.balance;`